conda/repodata.py

```diff
@@ -0,0 +1,99 @@
+# (c) Continuum Analytics, Inc. / http://continuum.io
+# All Rights Reserved
+#
+# conda is distributed under the terms of the BSD 3-clause license.
+# Consult LICENSE.txt or http://opensource.org/licenses/BSD-3-Clause.
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+from .core import repodata as _impl
+from .connection import CondaSession
+from os.path import join
```

Comment:
I would appreciate if we could document these rules somewhere (the wiki might be a good place), as not everyone uses the same tool(s) to edit the code. (I'm using emacs myself, and so would need to manually re-arrange the imports if I knew the rules.)